src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -441,6 +441,16 @@ def transform_request(self,
         if not filters:
             filters = {"file": {}}
         filters = filters['file']
+
+        translation_dict = request_config['translation']
+        for facet in filters:
+            if facet not in translation_dict:
+                raise BadArgumentException(f"Unable to filter by undefined facet, {facet}.")
+
+        sort_facet = pagination["sort"]
+        if sort_facet not in translation_dict:
```

Comment:
If `sort_facet` is not used anywhere else, I think it might actually be better to just use `pagination["sort"]` directly (unless you exceed 100 line characters, which I would doubt).

Reply:
It is used again, though. In the line below.

Consider just reusing the variable `facet`.